src/managers/TenorManager.ts

```diff
@@ -61,15 +60,18 @@ class TenorManager {
 	}
 
 	private praseResult(img: any): TenorImage {
-		const gif = img['media_formats']['gif'];
+		//Use tinygif object for compnent render. Tenor urls are the same betweent tinygif and gif
+		const tinyGif = img['media_formats'][MediaFilter.TINYGIF];
```

Comment:
if you remove this line and just use the next line, the media loads are in the MBs.

Reply:
Then save tinygifs in property named `preview` and use it for displaying previews in picker but still pass regular gif to callback

Reply:
I believe my latest update accomplishes this.